examples/1994_06_09_aerial_scg.ipynb

```diff
@@ -0,0 +1,383 @@
+      "text/plain": [
+       "<Figure size 1600x600 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "all_detections, all_scores, all_subpixel_scores = preproc.detect_fiducials(grid_size=7, max_workers=5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Remove wrong detection\n",
+    "\n",
+    "same as `friedrichknuth/hipp` we set None all miss fiducial detections with the `process_fiducials_detection` method. We validate a detection with 2 methods a geometric one and a score based method. The detection is valid if at least one validation is passed. The most important validation is the score based method. We calculate for each group of fiducials (\"corner_top_right\", ...) the median. Then we validate each detection with a score above the median with a treshold (0.1). The second validation is to compare each angle with 90 degree, if an angle is closed enough of 90 degree we validate the 3 points. This second step can be usefull to validate detection that have a low matching score but are still true. \n",
```

Comment:
I don't find the name "process_fiducials_detection" very intuitive. "Process" is too vague and "detection" implies that some detection is performed. Would "filter_fiducials" or "filter_detected_fiducials" be appropriate?

Reply:
Regarding the score threshold, is it done exactly the same as in Friedrich's HIPP? It is also not very clear, is the threshold "score > median + 0.1" or "score > median - 0.1"? When reading this text, I understand the first example, but the second example makes more sense.
Instead of using a fixed threshold (0.1), would it work with a factor of the std?